migrator/src/main/scala/com/scylladb/migrator/writers/DynamoDB.scala

```diff
@@ -23,27 +24,39 @@ object DynamoDB {
       if (readThroughput.isEmpty || writeThroughput.isEmpty) Some("25")
       else None
 
-    val renamedDf = renames
-      .foldLeft(df) {
-        case (acc, Rename(from, to)) => acc.withColumnRenamed(from, to)
+    val jobConf = new JobConf(spark.sparkContext.hadoopConfiguration)
+
+    def setOptionalConf(name: String, maybeValue: Option[String]): Unit =
```

Comment:
dedup if possible

Reply:
Done in [8e9728d](https://github.com/scylladb/scylla-migrator/pull/105/commits/8e9728d63ff2cdf79f5bb5195b607ba80e684cbe).